client/src/components/About.js

```diff
@@ -1,7 +1,27 @@
 import React from "react";
 
 const About = () => {
-  return <div>About</div>;
+  return;
+  <>
+    <div>About </div>
+    <div>
+      <p>
+        Maryhill Integration Network (MIN) SCIO was established in 2001 and
+        became a registered charity (SCO37300) in March 2006. We are a Scottish
+        Charitable Incorporated Organisation . We have a voluntary Board of
+        Trustees.
+      </p>
+
+      <p>
+        MIN brings refugee, migrant and local communities together through, art,
+        social, cultural and educational groups and projects, offering people a
+        chance to learn new skills, meet new people, share experiences and take
+        part in worthwhile activities to improve their lives and the life of
+        their communities.{" "}
```

Comment:
you left an extra `{" "}` - I don't think we need it - can you please remove it?